### ISSUE TITLE
Inherited view properties are not correctly assigned on some components

### ISSUE THREAD

**Issue by cayleyh** [shoutem/ui#567]
Inherited view props such as `testID` and `accessibilityLabel` are not being correctly passed along to the underlying component in some cases.

For example, the `Spinner` component does not do this, so code like this:

```
<Spinner accessibilityLabel="Loading" />
```

Does not add the accessibilityLabel prop to the underlying `ActivityIndicator` instance.

This is going to be an issue for basically any component that doesn't pass along all the properties to the underlying objects.

I didn't check too deeply, but there seems to be a mix. The simple components like Button that just defer to the underlying RN implementation seem to mostly do this correctly. Maybe `Spinner` was just missed.

The more complex components do not do any of this, but it's not clear with my limited understanding of the code base how these general properties should be applied from a top level, particularly with the accessibility ones.
